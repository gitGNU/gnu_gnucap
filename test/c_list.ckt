'play with the list command
V1   1  0  generator(1)
L3   1  2  1.
R4   2  0  1.
R5   1  3  .97
C6   3  0  1.
.option trace
.list
.list r*
.list l3 r5
.list l3 - r5
.list l3 -
.list l3
.list c6 -
.end
