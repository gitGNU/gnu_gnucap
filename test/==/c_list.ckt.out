'play with the list command 
list 
V1  1  0  GENERATOR
L3  1  2  1.
R4  2  0  1.
R5  1  3  0.97
C6  3  0  1.
.option trace 
list r* 
R4  2  0  1.
R5  1  3  0.97
list l3 r5 
L3  1  2  1.
R5  1  3  0.97
list l3 - r5 
L3  1  2  1.
R4  2  0  1.
R5  1  3  0.97
list l3 - 
L3  1  2  1.
R4  2  0  1.
R5  1  3  0.97
C6  3  0  1.
.option trace 
.list 
.list r* 
.list l3 r5 
.list l3 - r5 
list l3 
L3  1  2  1.
list c6 - 
C6  3  0  1.
.option trace 
.list 
.list r* 
.list l3 r5 
.list l3 - r5 
.list l3 - 
.list l3 
end 
